content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -1,135 +1,290 @@
+If your URL is a local directory path (e.g. beginning with a forward slash `/`
+such as `/home/cloned-git-repositories/my-git-repository.git`), you can proceed
+to click the *Create Pipeline* button.
 
-[.boxshadow]
-image:blueocean/creating-pipelines/github-personal-access-token.png['Creating a Blue Ocean GitHub Access Token']
+Blue Ocean will then scan your local repository's branches for a `Jenkinsfile`
+and will commence a Pipeline run for each branch containing a `Jenkinsfile`. If
+Blue Ocean cannot find any `Jenkinsfile`, you will be prompted to begin creating
+one through the <<pipeline-editor#,Blue Ocean Pipeline editor>>.
 
-=== Select a GitHub Account or Organization
 
-All repositories on Github are grouped by owner, either an account or organization.
-When creating Pipelines, Blue Ocean mirrors that structure,
-asking users to select an account or organization which owns the repositories
-from which it will add Pipelines.
+==== Remote repositories
 
-[.boxshadow]
-image:blueocean/creating-pipelines/github-org.png['Selecting an Account or
-Organization',width=60%]
+If your URL is for a remote Git repository, then as soon as you begin typing
+the URL, starting with either:
+
+* `ssh://` - e.g.
+  `ssh://gituser@server-url/git-server-repositories/my-git-repository.git` +
+  or
+* `user@host:path/to/git/repo.git` - e.g.
+  `gituser@server-url:git-server-repositories/my-git-repository.git`,
+
+Blue Ocean provides you with its own public SSH key for your current/logged in
+Jenkins user.
 
-From here, Blue Ocean offers two styles of Pipeline creation, either
-"<<github-new-pipeline, single Pipeline>>" or
-"<<github-auto-discover, discover all Pipelines>>".
+. Ensure that this public SSH key (which you can copy and paste from the Blue
+  Ocean interface) has been configured for the remote Git server's user account
+  (e.g. within the `authorized_keys` file of the machine's `gituser/.ssh`
+  directory) before continuing. +
+  *Note:* This process allows your Jenkins user account to access the
+  repositories that your Git server's user account (e.g. `gituser`) has access
+  to. Read more about this in
+  link:https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server[Setting
+  Up the Server] of the link:https://git-scm.com/book/en/v2/[Pro Git documentation].
+. When done, return to the Blue Ocean interface and click the *Create Pipeline*
+  button.
 
-[[github-new-pipeline]]
-==== New Pipeline from a Single Repository
+Blue Ocean will then scan your local repository's branches for a `Jenkinsfile`
+and will commence a Pipeline run for each branch containing a `Jenkinsfile`. If
+Blue Ocean cannot find any `Jenkinsfile`, you will be prompted to begin creating
+one through the <<pipeline-editor#,Blue Ocean Pipeline editor>>.
 
-Selecting "New Pipeline" allows the user select and create a Pipeline for
-a single Repository.
+[WARNING]
+====
+Since the Blue Ocean Pipeline editor saves edited Pipelines to Git repositories
```

Comment:
I think this is a bit too verbose to say "it only supports Git over SSH"

Reply:
The interface puzzles me a bit here ... When I type `https://`, the interface shows me ...

<img width="597" alt="only-ssh-supported" src="https://user-images.githubusercontent.com/23667807/34355508-eaa5d4ba-ea89-11e7-9f1a-04e76c085ef4.png">

... and the fact that I'm presented with a **Credentials** drop-down to choose any of my current Jenkins user's configured credentials (including the `jenkins-generated-ssh-key`) makes me think that more connection protocols than just 'Git over SSH' are supported for remote Git repositories.

@rtyler , if 'Git over SSH' is the _only_ connection protocol supported by Blue Ocean, then I would change the wording of this warning to indicate this. (Let me know and I'll create a ticket.)

Also, is there a reason why if I specify `https://git-server/some-git-repo.git`, I'm still presented with the **Credentials** drop-down?